config.toml

```diff
@@ -6,13 +6,13 @@ relativeURLs = true
 
 [params]
     email = "info@schulichignite.com"
-    #status = "recruiting" # When no recruitment is happening, and mailing list is available
-    status = "signup"     # When recruitment is happening
+    status = "recruiting" # When no recruitment is happening, and mailing list is available
```

Comment:
This should work for now, but we should have another PR to change it to live on session 0